rrfs-test/validated_yamls/templates/obtype_config/abi_g16.yaml

```diff
@@ -0,0 +1,481 @@
+  # Observation Space (I/O)
+  # -----------------------
+     - obs space:
+         name: abi_g16
+         distribution:
+           name: "@DISTRIBUTION@"
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: data/obs/ioda_abi_g16.nc
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: jdiag_abi_g16.nc4
+         io pool:
+           max pool size: 1
+         simulated variables: [brightnessTemperature]
+         observed variables: [brightnessTemperature]
+         channels: &abi_g16_channels 7-16
+ 
+  # Observation Operator
+  # --------------------
+       obs operator:
+         name: CRTM
+         Absorbers: [H2O, O3, CO2]
+         #Clouds: [Water, Ice]
+         #Cloud_Fraction: 1.0
+         SurfaceWindGeoVars: uv
+         obs options:
+           Sensor_ID:  abi_g16 
+           EndianType: little_endian
+           CoefficientPath: data/crtm/
+           IRVISlandCoeff: IGBP
+         linear obs operator:
+           Absorbers: [H2O, O3]
+  # Observation Bias Correction (VarBC)
+  # -----------------------------------
+       obs bias:
+         input file: data/obs/abi_g16.satbias.nc4
+         output file: out_abi_g16.satbias.nc4
+         variational bc:
+           predictors:
+           - name: constant
+           - name: lapseRate
+             order: 2
+             tlapse: &abi_g16_tlapse data/obs/abi_g16.tlapse.txt
+           - name: lapseRate
+             tlapse: *abi_g16_tlapse
+           - name: emissivityJacobian
+           - name: sensorScanAngle
+             var_name: sensorScanPosition
+             order: 4
+           - name: sensorScanAngle
+             var_name: sensorScanPosition
+             order: 3
+           - name: sensorScanAngle
+             var_name: sensorScanPosition
+             order: 2
+           - name: sensorScanAngle
+             var_name: sensorScanPosition
+         covariance:
+           minimal required obs number: 20
+           variance range: [1.0e-6, 10.0]
+           step size: 1.0e-4
+           largest analysis variance: 10000.0
+           prior:
+             input file: data/obs/abi_g16.satbias_cov.nc4
+             inflation:
+               ratio: 1.1
+               ratio for small dataset: 2.0
+           output file: out_abi_g16.satbias_cov.nc4
+  # Observation Filters (QC)
+  # ------------------------
+       obs prior filters:
+       - filter: Perform Action
```

Comment:
Can we have a short (one line) comment before each `- filter:` line? Example here would be `# Initial error assignment`

Reply:
Good suggestion, Thanks!  I will update it with comment line before each filter.